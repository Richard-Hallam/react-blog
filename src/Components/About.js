import React from "react";
import '../styles/About.css'

function About(){
    return(
        <p>I am richard and stuff</p>
    )
}


export default About;