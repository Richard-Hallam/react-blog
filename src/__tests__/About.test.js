import React from "react";
import { getByText, render, screen } from '@testing-library/react';
import About from "../Components/About";

describe("Projects", () => {
    const { asFragment } = render(<About />);

    it("renders correctly", ()=>{
        expect(asFragment()).toMatchSnapshot();
    });


});