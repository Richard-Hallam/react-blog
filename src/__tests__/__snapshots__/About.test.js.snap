// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders correctly 1`] = `
<DocumentFragment>
  <div
    class="place-self-center"
  >
    <div>
      <p
        class="text-base text-center"
      >
        I'm Richard Hallam
      </p>
      <p>
        I am a programmer on a continuous journey of learning and creating. During 2023 I successfully completed a 28 week bootcamp where I developed many skills, including React, Node.js, MySQL, JavaScript, and Test-Driven Development. This experience has greatly improved my web development and programming in general. I am currently seeking employement and pursuing various personal projects partially for fun as well as to further develop my skills.
      </p>
      <p
        class=" text-base text-center"
      >
        Here are my github contributions
      </p>
      <p
        class="flex justify-center"
      >
        <img
          alt="Richard-Hallam's Github chart"
          class="items-center"
          src="http://ghchart.rshah.org/Richard-Hallam"
        />
      </p>
      <p
        class="text-xs font-light text-center"
      >
        The above image was generated using the following api https://github.com/2016rshah/githubchart-api
      </p>
    </div>
    <div>
      <p
        class="text-center"
      >
        Richard Hallam
      </p>
      <p
        class="text-center"
      >
        Junior Developer | Tech Enthusiast | Constant Learner
      </p>
      <p
        class="text-base font-semibold text-center"
      >
        Email: richard.hallam@live.co.uk
      </p>
    </div>
  </div>
</DocumentFragment>
`;
